src/wing/CCPACSWingCell.cpp

```diff
@@ -95,34 +96,38 @@ namespace WingCellInternal
 using namespace WingCellInternal;
 
 CCPACSWingCell::CCPACSWingCell(CCPACSWingCells* parentCells, CTiglUIDManager* uidMgr)
-    : generated::CPACSWingCell(parentCells, uidMgr) {
+    : generated::CPACSWingCell(parentCells, uidMgr)
+{
     Reset();
 }
 
-CCPACSWingCell::~CCPACSWingCell() {}
+CCPACSWingCell::~CCPACSWingCell()
+{
+}
 
 void CCPACSWingCell::Invalidate()
 {
-    cache = boost::none;
+    geometryValid = false;
+    cache.valid   = false;
 }
 
 void CCPACSWingCell::Reset()
 {
-    m_uID = "";
-    
-    cache = boost::none;
+    m_uID         = "";
+    geometryValid = false;
+    cache.valid   = false;
 }
 
 bool CCPACSWingCell::IsConvex() const
 {
     Point2D p1, p2, p3, p4;
-    
+
     // calculate for all 4 edges the relative position of eta/xsi
     GetTrailingEdgeInnerPoint(&p1.x, &p1.y);
     GetTrailingEdgeOuterPoint(&p2.x, &p2.y);
-    GetLeadingEdgeOuterPoint (&p3.x, &p3.y);
-    GetLeadingEdgeInnerPoint (&p4.x, &p4.y);
-    
+    GetLeadingEdgeOuterPoint(&p3.x, &p3.y);
+    GetLeadingEdgeInnerPoint(&p4.x, &p4.y);
+
     // trailing edge
     bool s1 = sign(p3, p1, p2) > 0.;
```

Comment:
const

Reply:
@bernhardmgruber What do you mean here? What should be const?